contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -282,6 +283,18 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
         swapPath = newSwapPath;
     }
 
+    /**
+     * @notice Sets the new stakingRewards contract for this vault
+     * @param newStakingRewards is the address of the new stakingRewards contract
+     */
+    function setStakingRewards(address newStakingRewards)
+        external
+        onlyOwner
+        nonReentrant
```

Comment:
Hack to reduce bytecode size: since `nonReentrant` is a modifier that gets copied, we can safely remove it for this function because we assume the owner will not call it and cause reentrancy

Reply:
Removed here: https://github.com/ribbon-finance/ribbon-v2/pull/194/commits/8a823b5a3d08f85388002a012fec7db7adfa475f